package.json

```diff
@@ -80,6 +80,7 @@
     "build:watch": "webpack --config webpack.config.js --watch",
     "build:server": "ENVIRONMENT=production PUBLIC_PATH=/viewer/static/ webpack --config webpack.config.js",
     "build:analyze": "ENVIRONMENT=production ANALYZE_BUNDLE=true webpack --config webpack.config.js",
-    "local": "concurrently --kill-others \"npm run build:watch\" \"npm run serve\""
+    "local": "ENGINE_PATH=../engine/build/playcanvas.js EXTRAS_PATH=../engine/build/playcanvas-extras.js concurrently --kill-others \"npm run build:watch\" \"npm run serve\"",
```

Comment:
Specifying env vars like this won't work on Windows, right?

Reply:
I haven't a clue! :D

Reply:
Let me rephrase. This won't work on Windows. 🤣

Reply:
Any idea if this will fail or will silently be ignored on windows? If windows silently ignores then I'd say it's probably ok.

Looks like windows users must do `set ENGINE_PATH=blah' before running the command to get the same result.

Reply:
Just found this, will try it https://www.npmjs.com/package/cross-env

Reply:
That seems to work on mac and according to the docs should work elsewhere too. (Though this could do with testing on windows).